### PR TITLE
feat: Fix tooltip and add favicon

### LINKED ISSUE #30: Feature: Tooltip for Avatars
### Type of feature

Feature

### Current behavior

Each card has a list of contributors with no context of who they are. I would love to add a tooltip on wh

Group 2

similar to https://github.com/open-sauced/hot/issues/21

### Desired solution

This may be accomplished through a [react tooltip](https://www.npmjs.com/package/react-tooltip) library or similar. 

1. Add a tooltip on hover
2. Ensure the [cursor-pointer](https://github.com/open-sauced/hot/blob/732680ea21028f593f3c7c35e733b5cdbece70fd/src/components/SecondaryNav.astro#L5) is added with Tailwind 

tooltip

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### LINKED ISSUE #36: Feature: Add a favicon
### Type of feature

Feature

### Current behavior

currently no favicon present.

Screen Shot 2021-12-23 at 3 37 13 PM

### Desired solution

Screen Shot 2021-12-23 at 3 36 57 PM

This can be grabbed from https://github.com/open-sauced/open-sauced/blob/main/public/favicon.ico

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by kelvinsekx**
## What type of PR is this? (check all applicable)

- [x] ♻️ Refactor
- [x] ✨ Feature
- [x] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR adds 
- the opensauced favicon
- a tooltip to every profile image
- layout structure to present codebase

## Related Tickets & Documents

- fixes #36 
- fixes #30 
## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [x] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📜 contributing.md
- [ ] 📓 docs
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -40,5 +40,6 @@ module.exports = {
     'testing-library/await-async-query': 'error',
     'testing-library/no-await-sync-query': 'error',
     'testing-library/no-dom-import': 'off',
+    'linebreak-style': 0,
```
Comment (bdougie) [inline .eslintrc.js:43]: Curious why this was added @kelvinsekx.

**Comment by bdougie** [comment]
@kelvinsekx This looks good. You will need to fix the merge conflicts and fill out the description items for the PR. That context is important for future contributors and documentation.

File: package.json [hunk 2]
```diff
@@ -57,6 +57,14 @@
...
   },
   "dependencies": {
-    "open-graph-scraper": "^4.11.0"
+    "open-graph-scraper": "^4.11.0",
+    "prop-types": "^15.7.2",
+    "react": "^17.0.2",
+    "react-tooltip": "^4.2.21"
+  },
+  "eslintConfig": {
+    "globals": {
```
Comment (0-vortex) [inline package.json:66]: put this into `.eslintrc.js` please, the point of that file is that we don't use `package.json` :D

Comment (kelvinsekx) [reply package.json:66]: https://github.com/open-sauced/hot/pull/51#discussion_r777061498

We resolved this already. Check the conversation we had.

Comment (kelvinsekx) [reply package.json:66]: https://github.com/open-sauced/hot/pull/51#discussion_r777021962

Comment (0-vortex) [reply package.json:66]: @bdougie has 1337 notifications per day, he focused on the content that you should modify, in this case adding `window` to our `eslint.globals` configuration. It's still a bad practice to have that configuration in 2 places, simply put it's either in `package.json` or `.eslintrc.js` but not both. [open-sauced/open-sauced](https://github.com/open-sauced/open-sauced) is using [.eslintrc.js](https://github.com/open-sauced/open-sauced/blob/main/.eslintrc.js), would prefer we keep that convention 🍕

Comment (kelvinsekx) [reply package.json:66]: Gooosh 😔

**Comment by 0-vortex** [comment]
> # PR Compliance Checks
> Thank you for your Pull Request! We have run several checks on this pull request in order to make sure it's suitable for merging into this project. The results are listed in the following section.
> 
> ## Watched Files
> This pull request modifies specific files that require careful review by the maintainers.
> 
> Files Matched
> 
> * package.json

This requires an `npm install` to bring the changes from `package.json` into `npm-shrinkwrap.json`

**Comment by kelvinsekx** [comment]
> > # PR Compliance Checks
> > Thank you for your Pull Request! We have run several checks on this pull request in order to make sure it's suitable for merging into this project. The results are listed in the following section.
> > ## Watched Files
> > This pull request modifies specific files that require careful review by the maintainers.
> > Files Matched
> > 
> > * package.json
> 
> This requires an `npm install` to bring the changes from `package.json` into `npm-shrinkwrap.json`

Have not been around my laptop. When I'm in I will resolve that.

File: .eslintrc.js (hunk 1 above)
Comment (bdougie) [reply .eslintrc.js:43]: Going to remove this, since it seems unrelated and should not block this from being merged.
```suggestion
```

**Comment by bdougie** [comment]
Thanks for another great contribution @kelvinsekx

**Pull request merged**